tests/conftest.py

```diff
@@ -1,10 +1,21 @@
 from unittest.mock import patch
 import pytest
+import os
+from pathlib import Path
+import shutil
 
 from gretel_synthetics.config import LocalConfig
+from gretel_synthetics.train import annotate_training_data
 
+test_data_dir = os.path.join(str(Path(__file__).parent.resolve()), 'data')
```

Comment:
`str((Path(__file__).parent / 'data').resolve())`, or skip the `str` and use `pathlib` downstream